dashboards/Insight/Home_Dashboard.json

```diff
@@ -2430,7 +2430,7 @@
       "type": "stat"
     }
   ],
-  "refresh": "",
+  "refresh": "1m",
```

Comment:
is this required?

Reply:
All other dashboards have this interval.

Reply:
hm, I see it was 1m, before the latest changes:
https://github.com/percona/grafana-dashboards/blob/2d78426fd4be96d43c2c31aa2218a2af1131875a/dashboards/Insight/Home_Dashboard.json#L2774